pkg/tools/cli/loadtest/service/insert/insert.go

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package insert
+
+import (
+	"context"
+	"reflect"
+	"sync"
+
+	"github.com/vdaas/vald/internal/client"
+	"github.com/vdaas/vald/internal/errgroup"
+	"github.com/vdaas/vald/internal/errors"
+	"github.com/vdaas/vald/internal/safety"
+	"github.com/vdaas/vald/pkg/tools/cli/loadtest/assets"
+)
+
+type insert struct {
+	eg errgroup.Group
+
+	w client.Writer
+	c int
+	n string
+
+	req []*client.ObjectVector
+}
+
+func New(opts ...InsertOption) (i *insert, err error) {
+	i = new(insert)
+	for _, opt := range append(defaultInsertOpts, opts...) {
+		if err = opt(i); err != nil {
+			return nil, errors.ErrOptionFailed(err, reflect.ValueOf(opt))
+		}
+	}
+
+	i.eg = errgroup.Get()
```

Comment:
It might be better to set it with FOP.:thinking:

Reply:
I use same code in [agent](https://github.com/vdaas/vald/blob/master/pkg/agent/ngt/service/ngt.go#L161)

Reply:
ok! I think it is good.